Makefile

```diff
@@ -108,7 +108,7 @@ vet: ## Run go vet against code.
 
 .PHONY: test
 test: manifests generate fmt vet envtest ## Run tests.
-	KUBEBUILDER_CONTROLPLANE_START_TIMEOUT=2m KUBEBUILDER_CONTROLPLANE_STOP_TIMEOUT=2m KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test $(shell go list ./... | grep -v /tests/integration) -coverprofile cover.out
+	KUBEBUILDER_CONTROLPLANE_START_TIMEOUT=2m KUBEBUILDER_CONTROLPLANE_STOP_TIMEOUT=2m KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test $(shell go list ./... | grep -v controllers | grep -v /tests/integration) -coverprofile cover.out
```

Comment:
I think we might also want to consider excluding `pkg/lib/sidecars/test`, `tests` and since we can't easily ignore generated code also `api`. On the other hand, with such a configuration it is also very questionable whether we should keep the code coverage check.

Reply:
I think we need to discuss this code coverage check

Reply:
Code coverage check will be optional